.github/scripts/publish-deploy.sh

```
@@ -1,15 +1,15 @@
-do
-    case "${flag}" in
-        e) ENDPOINT=${OPTARG};;
-        p) PROJECTNAME=${OPTARG};;
-        o) ORG=${OPTARG};;
-        *) echo "Usage: $0 [-p projectname] [-o org] [-e endpoint]" && exit 1;;
-    esac
-done
-
-IPFSCID=$(npx subql publish -o -f .)
-
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)